src/runtime-rs/crates/resource/src/share_fs/utils.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::path::{Path, PathBuf};
+
+use anyhow::Result;
+use kata_sys_util::mount;
+
+use super::*;
+
+pub(crate) fn share_to_guest(
+    source: &str,
+    target: &str,
```

Comment:
Parameter `source` is an absolute path, and `target` is a relative path, they are different, should write some comments note that.

Reply:
Thanks for your review, already updated.